apps/studio/src/components/ResourceSelector/ResourceSelector.tsx

```diff
@@ -1,160 +1,190 @@
-import { Suspense, useState } from "react"
-import {
-  Box,
-  Flex,
-  HStack,
-  Icon,
-  Skeleton,
-  Spacer,
-  Text,
-} from "@chakra-ui/react"
-import { Button, Link } from "@opengovsg/design-system-react"
+import { Suspense } from "react"
+import { Box, Flex, Skeleton, Text, VStack } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
 import { ResourceType } from "~prisma/generated/generatedEnums"
-import { BiHomeAlt, BiLeftArrowAlt } from "react-icons/bi"
 
-import { trpc } from "~/utils/trpc"
-import { ResourceItem } from "./ResourceItem"
+import type { ResourceItemContent } from "~/schemas/resource"
+import { useSearchQuery } from "~/hooks/useSearchQuery"
+import { getUserViewableResourceTypes } from "~/utils/resources"
+import {
+  LoadingResourceItemsResults,
+  NoItemsInFolderResult,
+  ResourceItemsResults,
+  ZeroResult,
+} from "./ResourceSelectorContent"
+import {
+  BackButtonHeader,
+  HomeHeader,
+  LoadingHeader,
+  SearchResultsHeader,
+} from "./ResourceSelectorHeader"
+import { SearchBar } from "./SearchBar"
+import { useResourceStack } from "./useResourceStack"
+
+const FILE_EXPLORER_DEFAULT_HEIGHT_IN_REM = 17.5
```

Comment:
i think this  might not be the way to do it! instead, we should extend a chakra prop, which allows us to override props like this naturally.